packages/website/docs/introduction.md

```diff
@@ -3,6 +3,38 @@ id: introduction
 title: What is Autocomplete?
 ---
 
-import Draft from './partials/draft.md'
+Autocomplete is an open-source, production-ready JavaScript library for building autocomplete experiences.
```

Comment:
What does "production-ready" adds in this sentence? Isn't it implied?

Reply:
Hah, true. I'm taking from lingo we use in our own docs when describing our [API clients](https://www.algolia.com/doc/api-client/getting-started/what-is-the-api-client/javascript/?client=javascript), [IS libraries](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/), [Netlify plug-in](https://www.algolia.com/doc/tools/crawler/netlify-plugin/netlify-faq/#how-to-build-a-custom-user-interface), etc.

While implicitly understood, this explicitly states this is an officially supported, stable library.

That said, I'm not wedded to it. :)

Reply:
That's fair!